content/en/cosign/verify.md

```diff
@@ -9,8 +9,28 @@ The general verification format with the `cosign verify` command is as follows.
 ```shell
 $ cosign verify --key <key path>|<key url>|<kms uri> <image uri> 
```

Comment:
We should note that --key is optional here, so `cosign verify [--key <key path>|<key url>|<kms uri>] >image uri>`